scripts/dspublisher-init-templates/dsp-global.css

```
@@ -1,14 +1,20 @@
 header h3 a {
   display: flex;
   align-items: center;
-  font-size: 25px;
+  font-size: var(--docs-font-size-h1);
```

H1 size would be huge 😅 

```suggestion
  font-size: var(--docs-font-size-h4);
```